src/Jackett.Common/Definitions/hdfans.yml

```
@@ -0,0 +1,179 @@
+---
+id: hdfans
+name: HDFans
+description: "HDFans is a CHINESE Private Torrent Tracker for HD MOVIES / TV"
+language: zh-CN
+type: private
+encoding: UTF-8
+links:
+  - https://hdfans.org/
+
+caps:
+  categorymappings:
+    - {id: 401, cat: Movies, desc: "Movies/电影"}
+    - {id: 404, cat: TV/Documentary, desc: "Documentaries/纪录片"}
+    - {id: 405, cat: TV/Anime, desc: "Animations/动漫"}
+    - {id: 402, cat: TV, desc: "TV Series/剧集"}
+    - {id: 403, cat: TV, desc: "TV Shows/综艺"}
+    - {id: 404, cat: Other, desc: "Education/教育"}
+    - {id: 406, cat: Audio/Video, desc: "MusicVideo/音乐MV"}
+    - {id: 407, cat: TV/Sport, desc: "Sports/运体"}
+    - {id: 409, cat: Other, desc: "Misc/其他"}
+    - {id: 408, cat: Audio, desc: "HQ Audio/无损音乐"}
+    - {id: 419, cat: Other, desc: "Software/软件"}
+    - {id: 421, cat: Other, desc: "Games/游戏"}
```

Please find the most appropriate categories - https://github.com/Jackett/Jackett/wiki/Jackett-Categories
Suggestion below:
```suggestion
    - {id: 419, cat: PC/0day, desc: "Software/软件"}
    - {id: 421, cat: PC/Games, desc: "Games/游戏"}
```

---

Please review your categories as they are not a perfect match for what is on the site.
I found the following in the **User CP > Tracker Settings** page:
401 Movies/电影
402 TV Series/电视剧"
403 Documentaries/纪录片
406 Music/音乐
416 TV Shows/综艺
417 Animations/动漫、动画
407 Music Videos/音乐视频
408 Concert/演唱会
404 Education/教育
405 Audio Books/有声读物
409 Drama/戏剧
418 Sports/体育
419 Software/软件
421 Games/游戏
423 E-Books/电子书
410 Others/其他